### ISSUE TITLE
Not linking symbol references

### ISSUE THREAD

**Issue by quekshuy** [tgreyuk/typedoc-plugin-markdown#77]
I'm using typedoc-plugin-markdown (version 2.1.1) with typedoc version 0.15.0. 
Symbol references (double square brackets) are no longer rendering as links. Is there any information I can provide that can help figure out why this is happening?

Thanks for the great work!

**Comment by tgreyuk** [comment]
@quekshuy I can't replicate using basic examples. I don't suppose you have a code snippet to share?

**Comment by quekshuy** [comment]
Sure.

__tsconfig.json__
```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "lib": ["esnext", "dom"],
    "declaration": true,
    "outDir": "lib",

    "strict": true,
    "moduleResolution": "node",
    "types": ["jest", "node"],
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true             
  },
  "include": [
    "src"
  ],
  "exclude": [
  ],
  "typedocOptions": {
    "mode": "modules",
    "out": "./docs",
    "includeDeclarations": true,
    "excludeExternals": true,
    "excludePrivate": true,
    "excludeProtected": true,
    "toc": true,
    "hideGenerator": true,
    "theme": "markdown",
    "readme": "./INTRO.md"
  }
}

```

__src/index.ts__

```ts
import { Wall } from './types'
/** 
 * Use a [[Wall]] to do something.
 * @param w The [[Wall]] that we use to do something.
 * @returns [[Wall]] that something was done on.
 */
export function myFunction(w: Wall): Promise<Wall> {
  return Promise.resolve(w)
}
```

__src/types.ts__

```ts
/**
 * Representation of a Wall returned from our API.
 */
interface Wall {
  /** Identifier for the wall */
  readonly id: number
}

export {
  Wall,
}
```

__INTRO.md__ _(used to generate the README)_

```markdown
# Intro

My objects
- [[Wall]]
```

Here are screenshots of the markdown that was generated:

__docs/README.md__
![image](https://user-images.githubusercontent.com/58797/63227106-bb0ffe00-c215-11e9-8bb6-1e60cb4b72b8.png)

The reference to `Wall` is not linked in the README.

__docs/_index_.md__
![image](https://user-images.githubusercontent.com/58797/63227111-da0e9000-c215-11e9-9221-3d2f7e92ab77.png)

The reference to `Wall` in the `@returns` description is not linked.

I'm sorry if I wasn't being descriptive enough. I was assuming that symbol references would be linked in all cases but it's just not happening in some cases. Would appreciate any feedback as to what I'm doing wrong here.

_Note: do let me know if the code sample should be submitted as a gist._

**Comment by tgreyuk** [comment]
@quekshuy Thanks for the detailed description which was very useful. The symbol references should now be fixed in the readme and return type/description. 

Available in 2.1.3. Let me know if you have any further issues.
